docs/src/pages/[platform]/components/collection/useCollectionProps.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export const useCollectionProps = (defaultValues) => {
+  const [direction, setDirection] = React.useState(defaultValues.direction);
+  const [gap, setGap] = React.useState(defaultValues.gap);
+  const [wrap, setWrap] = React.useState(defaultValues.wrap);
+
```

Comment:
Plz add the logic to preserver demo state

Reply:
thanks the demo state has been added